newIDE/app/src/AssetStore/PrivateAssetPackDialog.js

```diff
@@ -0,0 +1,244 @@
+  'audio',
+  'partial',
+];
+
+const contentTypeToMessageDescriptor = {
+  sprite: t`Sprites`,
+  '9patch': t`Panel sprites`,
+  tiled: t`Tiled sprites`,
+  particleEmitter: t`Particle emitters`,
+  font: t`Fonts`,
+  audio: t`Audios`,
+  partial: t`Other`,
+};
+
+const PrivateAssetPackDialog = ({
+  privateAssetPack: { id, name, description, sellerId },
+  onClose,
+}: Props) => {
+  const [
+    assetPackDetails,
+    setAssetPackDetails,
+  ] = React.useState<?PrivateAssetPackDetails>(null);
+  const [isFetchingDetails, setIsFetchingDetails] = React.useState<boolean>(
+    false
+  );
+  const [
+    openSellerPublicProfileDialog,
+    setOpenSellerPublicProfileDialog,
+  ] = React.useState<boolean>(false);
+  const [
+    sellerPublicProfile,
+    setSellerPublicProfile,
+  ] = React.useState<?UserPublicProfile>(null);
+  const [errorText, setErrorText] = React.useState<?React.Node>(null);
+  const windowWidth = useResponsiveWindowWidth();
+
+  React.useEffect(
+    () => {
+      (async () => {
+        setIsFetchingDetails(true);
+        try {
+          const details = await getPrivateAssetPackDetails(id);
+          const profile = await getUserPublicProfile(sellerId);
+          setAssetPackDetails(details);
+          setSellerPublicProfile(profile);
+        } catch (error) {
+          if (error.response && error.response.status === 404) {
+            setErrorText(
+              <Trans>
+                Asset pack not found - An error occurred, please try again
+                later.
+              </Trans>
+            );
+          } else {
+            setErrorText(<Trans>Unknown error</Trans>);
+          }
+        } finally {
+          setIsFetchingDetails(false);
+        }
+      })();
+    },
+    [id, sellerId]
+  );
+
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <>
+          <Dialog
+            maxWidth="lg"
+            open
+            onRequestClose={onClose}
+            actions={[
+              <TextButton
+                key="cancel"
+                label={<Trans>Cancel</Trans>}
+                onClick={onClose}
+              />,
+            ]}
+            onApply={() => {}}
+            flexColumnBody
+            fullHeight
+          >
+            {errorText ? (
+              <AlertMessage kind="error">
+                <Text>{errorText}</Text>
+              </AlertMessage>
+            ) : isFetchingDetails ? (
+              <>
+                <Text size="title">{name}</Text>
+                <Column expand>
+                  <PlaceholderLoader />
+                </Column>
+              </>
+            ) : assetPackDetails && sellerPublicProfile ? (
+              <>
+                <Column noMargin>
+                  <Text size="title">{name}</Text>
+                  <Text size="body2">
+                    <Trans>by</Trans>{' '}
+                    <Link
+                      onClick={() => setOpenSellerPublicProfileDialog(true)}
+                      href=""
```

Comment:
Looks like Link has never been used without an href. 
Are we sure it works fine on web?

Reply:
I didn't notice any issue with using that. I will change it for "#" so that React does not translate this as the following html:
```
<a ... href>...</a>
```